public/services/twitter.service.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+angular.module('textAnalysis')
+    .factory('twitterService', ['$q', '$location', '$http',
+        function($q, $location, $http) {
+
+            var userTimeline;
+
+            return {
+
+                loadTweets: function(userInput) {
+                    var deferred = $q.defer();
+                    $http.get('/api/' + userInput)
+                        .success(function(data) {
+                            deferred.resolve(data);
+                            userTimeline = data;
+                            console.log(userTimeline);
```

Comment:
Careful committing `console.log`, this will cause errors in some versions of Internet Explorer and break your app.  If you're looking to add logging to your app the best thing to do is inject `$log` into your factory/controller/etc and use that.  Something like:

```
+angular.module('textAnalysis')
    .factory('twitterService', ['$q', '$location', '$http', '$log',
        function($q, $location, $http, $log) {
```

```
$log.log(userTimeline);
```

The `$log` module is a drop in replacement for `console`, they work the same exact way, but Angular gracefully regresses in browsers which do not support `console` and your app doesn't break.  For more info check out https://docs.angularjs.org/api/ng/service/$log